include/roboteam_ai/world_new/World.hpp

```diff
@@ -79,7 +79,7 @@ class World {
      *
      * Undefined behavior may occur if feedback is used after being passed to this function
      */
-    void updateFeedback(uint8_t robotId, proto::RobotFeedback &feedback);
+    void updateFeedback(std::unordered_map<uint8_t, proto::RobotFeedback> feedback);
```

Comment:
Why this change? Do we do batch updating now?

Reply:
At the time this was the quickest fix, but we can change to id-based updating if you like. That would mean that there will be a loop in application manager that will update 11 robots on id, aka looking up the mapped value in IOManager and using that as input for the map that is in World